shlr/spp/spp.h

```diff
@@ -12,7 +12,7 @@
 int r_sys_setenv(const char *key, const char *value);
 #endif
 
-#ifdef __WINDOWS__
+#if defined (_MSC_VER) || (__WINDOWS__)
```

Comment:
@radare - no, __WINDOWS__ is defined also for Mingw32, Cygwin and MSYS2 (Mingw-w64), while _MSC_VER is defined only for MSVC.